app/assets/stylesheets/custom.scss

```diff
@@ -148,10 +148,10 @@ a {
 }
 
 .profile-header {
-    background-color: white;
-    padding-top: 60px;
-    padding-left: 50px;
-    padding-bottom: 40px;
+  background-color: white;
+  padding-top: 60px;
```

Comment:
Properties should be ordered background-color, padding-bottom, padding-left, padding-top